### DISCUSSION TITLE
Welcome to safecastapi Discussions!

### DISCUSSION THREAD

**Discussion by robouden** [Safecast/safecastapi#748] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lugulbanda** [comment]
since today i noticed that i cant access the Gatway from https://api.safecast.org/en-US

loading is very slow and i cant go over the starting page
 
this leads me to what i wrote a few years ago,

it makes no sence that i have to start with all worldwide uploaded api data, move from there to imports from everyone, and then to mine ..
needles Traffic needless Datatransfer, why cant i just jump directly to my data, if i use the webpage to edit the metadata

instead iam forced to see all the other parts over a few steps wich are not needed, if i just want to add some metadata, overthe webpage..
typing it into my smartphone is not working good cuz my screen is to small, even if i can there just jump to my data directly...

please update the API Page in 2 Ways
remember who iam (credits, height, direction) 
and if logged in after sign in, move to my private user page with new prozessed data without metadata...

question, safecast is using the google map, were u can search and find citys by name and postal adress..
if i use the route planer on google it shows all Citys along a route, why cant this be implemented into the api metadata?
cuz entering all the citys and towns visited is the most work on the metadata...


saves me time to hover around a slow gateway.. wich i try to connect now since halfe a day allways loosing connection ..

https://api.safecast.org/en-US/bgeigie_imports
504 Gateway Time-out
nginx/1.20.0

Diese Seite funktioniert nichtapi.safecast.org kann diese Anfrage momentan nicht verarbeiten.
HTTP ERROR 503''

i tryed to ping the server from various locations but none of them gets through 

i also noticed that all my measurements from last 2 weeks didnt found the way into the Tile map.
i remember it can be up to 3 Days to update the map....

and without the working api i cant even access the measured travels cuz i dont have the numbers.


PS: if my english sounds strange iam sorry its not my native Language ;-)

**Reply by robouden** [reply]
Thanks for writing the issue down. Sometimes, when the server is creating the 4-hour backup for the CSV export, the site is not that fast. If you have a better setup, please share that with us and show how this can/should be done better.
